litestar/middleware/session/client_side.py

```diff
@@ -206,6 +206,9 @@ async def load_from_connection(self, connection: ASGIConnection) -> dict[str, An
                 return self.load_data(data)
         return {}
 
+    def get_session_id(self, connection: ASGIConnection) -> str | None:
```

Comment:
It implements the abstract method of `BaseSessionBackend` which expects a `str` for server-side backend and `None` for client-side backend. This is why I kept the signature the same, even though it will never return a string.